src/content/docs/support/faq.mdx

```diff
@@ -170,7 +170,88 @@ sudo rm -r /var/cache/pacman/pkg/*
 sudo pacman -S nvidia
 ```
 
-## Cachy Browser
+## Cachy Browser (Deprecated)
+
+### Migrating Your Profile from Cachy Browser to Firefox
+
+Cachy Browser is based on Firefox (via LibreWolf), so migrating your profile, including history, bookmarks, extensions, and settings, to the standard Firefox is generally possible by copying the profile directory. However, always back up your data before proceeding.
+
+There are a few methods, ranging from a full profile copy to using sync or manual transfer.
+
+**Important First Step: Back Up Your Profiles!**
+
+Before you start, it's crucial to back up both your Cachy Browser profile and your Firefox profile (if you have existing data you want to keep). This way, you can restore them if something goes wrong. Simply copy the entire profile folders (locations described below) to a safe place.
+
+#### Method 1: Direct Profile Copy
+
+This method attempts to copy *everything* from your Cachy Browser profile to Firefox. It's the most likely way to transfer all settings and extension data but carries a small risk of minor incompatibilities.
+
+1.  **Locate Your Cachy Browser Profile Folder:**
+    *   Open Cachy Browser.
+    *   Type `about:profiles` in the address bar and press Enter.
+    *   Find the profile currently in use (it usually says "This is the profile in use...").
+    *   Click the "Open Directory" (or similar wording like "Show in Finder" on macOS, "Open Folder" on Windows/Linux) button next to the "Root Directory". This will open the folder containing your profile data.
```

Comment:
lets keep only "Open Directory", as thats what Cachy Browser has on CachyOS

no need for "Show in Finder" and "Show in Finder"